README.md

```diff
@@ -0,0 +1,3 @@
+# fangs
+
+A library that makes integrating Cobra and Viper easier and more consistent.
```

Comment:
this is a good opportunity to add an example of how this is used and a few points as to why using this makes it easier to use Cobra and Viper together.